doc/sphinx/source/quickstart/installation.rst

```diff
@@ -393,4 +393,12 @@ confirm that no errors are reported:
 
 .. code-block:: bash
 
-    python setup.py test
+    pytest -m "not installation"
```

Comment:
If you want to use the pytest command, you'll also need to install the test dependencies, so the command `pip install -e .` above would need to be changed to `pip install -e .[develop]` (or `pip install -e .[test]`if you want minimal extra dependencies, but I think most people interested in installing from source will want the other development tools as well.

Reply:
yes, so what do you recommend? I always use `-e .[develop]` so I didn't even think of thinking :grin:

Reply:
That would be fine, it just needs to be updated in the text above